integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -200,8 +192,17 @@ class SaucelabsSession {
 
   async getSessionId() {
     if (this.sessionId === undefined) {
-      const session = await this.driver.getSession();
-      this.sessionId = session.getId();
+      if(isMobileDomain(this.domain))
+      {
+        const cap = await this.driver.getCapabilities();
+        const url = cap.get('testobject_test_report_api_url')
```

Comment:
I thought testobject is deprecated?

Reply:
It's not exactly that testObject has been completely deprecated, but rather now it runs under the hood of the saucelabs api. The url it returns is the normal saucelabs test url https://app.saucelabs.com/tests/<JobID> this is the method provided by saucelabs support to grab the jobid.